lib/suspenders/app_builder.rb

```diff
@@ -531,16 +512,12 @@ def raise_on_missing_translations_in(environment)
       uncomment_lines("config/environments/#{environment}.rb", config)
     end
 
-    def override_path_for_tests
-      if ENV['TESTING']
-        support_bin = File.expand_path(File.join('..', '..', 'spec', 'fakes', 'bin'))
-        "PATH=#{support_bin}:$PATH"
-      end
+    def run_heroku(command, environment)
+      run "heroku #{command} --remote #{environment}"
```

Comment:
Yes, my idea is to eventually get it all removed. This is just a start.
On Thu, 5 Nov 2015 at 02:57, Tute Costa <notifications@github.com> wrote:

> In lib/suspenders/app_builder.rb
> <https://github.com/thoughtbot/suspenders/pull/646#discussion_r43971619>:
>
> > @@ -531,16 +512,12 @@ def raise_on_missing_translations_in(environment)
> >        uncomment_lines("config/environments/#{environment}.rb", config)
> >      end
> >
> > -    def override_path_for_tests
> > -      if ENV['TESTING']
> > -        support_bin = File.expand_path(File.join('..', '..', 'spec', 'fakes', 'bin'))
> > -        "PATH=#{support_bin}:$PATH"
> > -      end
> > +    def run_heroku(command, environment)
> > +      run "heroku #{command} --remote #{environment}"
>
> There are many heroku specific snippets in the AppBuilder. What is the
> boundary, you think? Should we continue extracting functionality into the
> new object?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/suspenders/pull/646/files#r43971619>.
>